README.md

```diff
@@ -1 +1,71 @@
+
+| Model Size | Hardware    | Sharding Strategy | Activation Checkpointing | Batch Size | Training Throughput <br/> (128 GPUs) | Profile Trace                                                         | 
+|------------|-------------|-------------------|--------------------------|------------|--------------------------------------|-----------------------------------------------------------------------|
+| 7b         | 128 * A100  | HSDP              | False                    | 2          | **3760 token/gpu/sec**               | [7b trace](https://ibm.box.com/s/ohaliqku0rl52jc9dhw1cb04opgssgy3)    |
+| 13b        | 128 * A100  | HSDP              | False                    | 1          | **1700 token/gpu/sec**               | [13b trace](https://ibm.box.com/s/2j0uib7m1p5wqjhv9dagq4331n62iyv6)   |
+| 34b        | 128 * A100  | FSDP              | True                     | 8          | **772 token/gpu/sec**                | [34b trace](https://ibm.box.com/s/tf7x6254egzgzrn6ceh6kgdgy0rbowz6)   |  
+| 70b        | 128 * A100  | FSDP              | True                     | 6          | **380 token/gpu/sec**                | [70b trace](https://ibm.box.com/s/5o1ohr1144nloqjrelsvunrq0rutyynu)   |
+
+
+## Installation
+You need to install the required packages by running the following command.
+We recommend running the latest [PyTorch nightlies](https://pytorch.org/) and latest [ibm-fms](https://github.com/foundation-model-stack/foundation-model-stack).
+```bash
+pip install -r requirements.txt
+```
+
+## Train
+
+### Model
+We trained one model, a replica of Llama2 7B as an exemplar on IBM curated data, the [Blue Pile](https://mitibmwatsonailab.mit.edu/research/blog/creating-space-for-the-evolution-of-generative-and-trustworthy-ai/). This model was trained to 2.2T tokens with a 4k context length on 128 A100 GPUs for a total of 162k GPU hours, achieving an efficiency of 3500 tokens/sec/GPU (~38B tokens/day), which is roughly 20% faster than the Llama2 published training time. These speedups were possible by combining multiple techniques - SDPA Flash v2 implementation, FSDP with overlap in computation and communication, and selective activation checkpointing.
+The generated model has a good performance on various metrics as evaluated by [lm-evaluation-harness](https://github.com/EleutherAI/lm-evaluation-harness), with MMLU score of 0.5. We share further [scores](docs/evaluation.md) in the details of the model for completeness.
+
+### Dataset
+We use an internally curated dataset, Blue Pile for training the model. We use sampling ratios similar to what Llama1 paper proposed with minor changes (e.g., no C4 dataset). Since the goal of this repo is to demonstrate the feasibility of training using PyTorch components at scale, we omit the details of the sampling ratios. The overall dataset is roughly 1.5T tokens and the model has seen all the tokens in the dataset at least once.
```

Comment:
maybe an additional `,` after "Blue Pile" or could reword to "For training the model we used ... called blue pile"